lang/es_ES.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -14494,3 +14494,9 @@ msgstr ""
 
 msgid "See Documentation"
 msgstr ""
+
+msgid "Services will not be affected because you don't have sufficient permission"
+msgstr ""
+
+msgid "Hosts will not be affected because you don't have sufficient permission"
+msgstr ""
```

Comment:
```suggestion
msgid "Services will not be affected because you don't have sufficient permission"
msgstr ""

#~ msgid "Hosts will not be affected because you don't have sufficient permission"
#~ msgstr ""
```

Reply:
Comment missing translations to avoid to display empty strings instead of english